hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/storefiletracker/StoreFileListFilePrettyPrinter.java

```diff
@@ -96,9 +99,7 @@ private void init() {
   public boolean parseOptions(String[] args) throws ParseException, IOException {
     HelpFormatter formatter = new HelpFormatter();
     if (args.length == 0) {
-      formatter
-        .printHelp("sft [--file=</path/to/tracker/file> | --table=<namespace:tablename|tablename>"
-          + " --region=<regionname> [--columnFamily=<columnfamily>] ]", options, true);
+      formatter.printHelp(helpMsg, options, true);
```

Comment:
The `printHelp` expects the command name as the first parameter and will generate the help message based on the passed Options. You don't need to include the parameters in the `cmdLineSyntax`.

Reply:
Fixed